<ui version="4.0" >
 <class>SQLMatchingDialog</class>
 <widget class="QDialog" name="SQLMatchingDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>340</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>490</width>
    <height>340</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>SQL Matching</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QFrame" name="frame" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string/>
        </property>
        <property name="pixmap" >
         <pixmap resource="../audex.qrc" >:/buttons/images/warning.png</pixmap>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <layout class="QVBoxLayout" >
        <item>
         <widget class="QLabel" name="label" >
          <property name="font" >
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text" >
           <string>Warning! Following entries found in the database:</string>
          </property>
          <property name="scaledContents" >
           <bool>false</bool>
          </property>
          <property name="wordWrap" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="listWidget" >
          <property name="editTriggers" >
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton" >
       <property name="text" >
        <string>Continue anyway</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../audex.qrc" />
 </resources>
 <connections/>
</ui>
